com.github.rizinorg.Cutter.yml

```diff
@@ -0,0 +1,29 @@
+#flatpak-builder --force-clean --user --install ./build ./com.github.rizinorg.cutter.yml
+app-id: com.github.rizinorg.Cutter
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: cutter
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+modules:
+  - name: cutter
+    buildsystem: cmake
```

Comment:
Usually it's better to use `cmake-ninja`, unless it doesn't work.

Reply:
The project does not use ninja in its build/install process. Why should I use it ?

P.S: I am not used to ninja

Reply:
We use ninja. :wink:  Using cmake generated ninja project is almost the same using cmake generated make files except you call ninja instead of make. The motivation for using ninja is that the build is often faster.

Reply:
Oh my bad 😅

Reply:
that's the idea, yes.